packages/near-membrane-base/src/intrinsics.ts

```diff
@@ -120,10 +127,21 @@ const ESGlobalsAndReflectiveIntrinsicObjectNames = toSafeArray([
     ...ReflectiveIntrinsicObjectNames,
 ]);
 
+function getGlobalObjectOwnKeys(source: object): PropertyKey[] {
+    const ownKeys = ReflectOwnKeys(source);
+    // WKWebView incorrectly excludes the 'webkit' own property of the global
+    // object from `Object.keys()` and `Reflect.ownKeys()` results, so add it.
+    // istanbul ignore if: currently unreachable via tests
+    if (ObjectHasOwn(source, 'webkit') && !ReflectApply(ArrayProtoIncludes, ownKeys, ['webkit'])) {
+        ownKeys[ownKeys.length] = 'webkit';
```

Comment:
If order doesn't matter, could just use `ownKeys.push('webkit')`

Reply:
`ownKeys[ownKeys.length] = 'webkit'` is equiv to `push` without the prototypal access.